.stylelintignore

```diff
@@ -1,2 +1 @@
 src/styles/normalize.scss
```

Comment:
Why is this getting deleted?

Reply:
I fixed the commit history so this shouldn't be in this PR anymore.